src/lib/server/ApiClient.ts

```diff
@@ -0,0 +1,5 @@
+export async function getUploadRequest() {
```

Comment:
The function should return ready to use data or throw an error.

Please add return types for this function.
See [this section](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-aliases) of typescript documentation

You can see API response example [here](https://github.com/S1ckret-Labs/family-archive-docs/blob/main/requirements/containers/backend/family-archive-web-server.md#get-upload-requests). It returns array of custom objects.

So define a new type `UploadRequest` which will describe the object.

Also your fetch request can fail. Please learn about error handling.

Reply:
Ok, i learn how to handle, and maybe i need to show users error? or its just for us? becouse i only show errors in console, but now it used try/catch construction, and aded custom type for request